libcloud/compute/drivers/runabove.py

```diff
@@ -68,14 +62,12 @@ def __init__(self, key, secret, ex_consumer_key=None):
 
     def list_nodes(self, location=None):
         """
-        List all Linodes that the API key can access
+        List all nodes.
 
-        This call will return all Linodes that the API key in use has access
-        to.
-        If a node is in this list, rebooting will work; however, creation and
-        destruction are a separate grant.
+        :keyword location: Location (region) used as filter
+        :type    location: :class:`NodeLocation`
 
-        :return: List of node objects that the API key can access
+        :returns: List of node objects
```

Comment:
I think it's just `return` (at least that's what most other code uses). It might also be that both works, but `return` is preferred for consistency reasons.